### ISSUE TITLE
Replace tooltips with react-tooltip-lite

### ISSUE THREAD

**Issue by MartijnHols** [WoWAnalyzer/WoWAnalyzer#2883]
I want to move to https://github.com/bsidelinger912/react-tooltip-lite. This allows us to finally have React components and regular elements in the tooltips without using the unsafe HTML attribute.

There's not a lot of work needed to implement this (basically just add the library to the `package.json` and start using it), but all of the existing tooltips would have to be migrated to prevent the debt and size consumption of two tooltip libraries.

This can be done in either `master` or `next`. `master` might be more troublesome since you might have to update whenever a PR is merged, while `next` will be more troublesome for *me* since I'll have to do that before releasing it :')

tl;dr:
1. add the new library. 
2. replace all data-tips with the new library. Convert all the strings with HTML in them to JSX.
3. Remove the `react-tooltip` dependency

**Comment by sMteX** [comment]
I'lll try to look at it in upcoming days

**Comment by MartijnHols** [comment]
Thanks @sMteX! Please don't forget to claim the bounty (it's prepaid).
